src/PlaneWaveBasis.jl

```diff
@@ -376,7 +399,8 @@ end
 The list of ``G + k`` vectors, in reduced coordinates.
 """
 function Gplusk_vectors(basis::PlaneWaveBasis, kpt::Kpoint)
-    map(G -> G + kpt.coordinate, G_vectors(basis, kpt))
+    coordinate = kpt.coordinate  # Avoid closure on kpt (not isbits)
```

Comment:
prefix with GPU: someone reading this has no idea why not being isbits is a problem

Reply:
Rather accelerator (to be more general)

Reply:
You know how I feel about being general before we actually know what we're generalizing to, but OK :-p